java/ql/src/Likely Bugs/Frameworks/JUnit/JUnit5MissingNestedAnnotation.md

```diff
@@ -0,0 +1,45 @@
+## Overview
+
+JUnit tests are grouped in a class, and starting from JUnit 5 users can group the test classes in a bigger class so they can share the local environment of the enclosing class. While this helps to organize the unit tests and foster code reuse, if an inner test class is not annotated with `@Nested`, the unit tests in it will fail to execute during builds.
```

Comment:
Adding a comma to help readability, and suggesting we use `larger` rather than `bigger` (less familiar).
```suggestion
JUnit tests are grouped in a class, and starting from JUnit 5, users can group the test classes in a larger class so they can share the local environment of the enclosing class. While this helps organize the unit tests and foster code reuse, if an inner test class is not annotated with `@Nested`, the unit tests in it will fail to execute during builds.
```